api/src/resolvers/wish.js

```diff
@@ -0,0 +1,26 @@
+const Wish = require('../db/Wish')
+
+module.exports = {
+  Query: {
+    wishes: async () => {
+      const wish = await Wish.find();
+      console.log(wish);
+      return wish;
+    },
+    wish: async (parent, { id }) => {
+      return await Wish.findById(id);
```

Comment:
waiting here necessary?

Reply:
yes it is, because this is the database lookup.

Reply:
I believe `await` is implied on a return from an async function. I don't think it's needed.